docs/UserGuide.md

```diff
@@ -68,7 +68,7 @@ Now that you have started **_TBM_**,
 
 The following section outlines the commands supported by **_TBM_**. A short description is given for each command along with the format that they should be executed with. Complex commands also come with examples to demonstrate the purpose that the command serve.
 
-Certain commands require parameters and the summary of these parameters as well as their prefixes and constraints can be found in our [parameters summary table](#parameter-constraints-summary)
+Certain commands require parameters and the summary of these parameters as well as their prefixes and constraints can be found in our [parameters summary table](#parameter-constraints-summary). **Note that parameter prefixes are special keywords and you should not use them for any other purposes except for indicating the start of a parameter.**
```

Comment:
Maybe add some examples like so: "Note that parameter prefixes such as `t/` or `n/` are special keywords..."